tensorflow_io/core/kernels/ffmpeg_kernels.cc

```diff
@@ -300,7 +302,7 @@ class FFmpegAudioStream : public FFmpegStream {
 
   Status OpenAudio(int64 index) {
     TF_RETURN_IF_ERROR(Open(AVMEDIA_TYPE_AUDIO, index));
-    TF_RETURN_IF_ERROR(OpenCodec());
+    TF_RETURN_IF_ERROR(OpenCodec(FF_THREAD_SLICE, 1));
```

Comment:
@kyamagu can you please add a comment mentioning that `FF_THREAD_SLICE` has a defined value of 2? I see that it is coming from `#include "libavcodec/avcodec.h"` but a comment will be helpful for future references.

Reply:
Sure